### PR TITLE
Fix several issues with exporting to tardis

### PR DESCRIPTION
**Pull request by yeganer**
This PR addresses several issues that prevent exporting data to tardis. These were mainly SQLOperationErrors.
Exporting to tardis with the database `carsus-medium.db` from carsus-db as input works on my machine and tardis produces almost bit identical output (except for some noise).

### CODE REVIEW

File: carsus/io/output/tardis_.py [hunk 1]
```diff
@@ -505,44 +505,37 @@ def get_energies(method):
...
-                raise AtomDataUnrecognizedMediumError(
-                    "The medium {} is not recognized (0 - vacuum, 1 - air)".format(wavelength.medium)
+        lines_q = (
+                self.session.
+                query(
+                    Line.line_id,
+                    Line.lower_level_id,
+                    Line.upper_level_id,
+                    wavelength._value.label('wavelength'),
+                    gf._value.label('gf'),
```
Comment (mishinma) [inline carsus/io/output/tardis_.py]: You should use `.quantity` accessor to query the value, so it should be `gf.quantity.value` (although it doesn't matter much here. With `.quantity` you are also able to convert quantities in queries, like:
```
level, level_energy_value = foo_session.query(
    Level, 
    LevelEnergy.quantity.to(u.Unit("cm-1"),  equivalencies=u.spectral()).value
)
```

Comment (yeganer) [inline carsus/io/output/tardis_.py]: @mishinma This did not work for me. That's why I resorted to querying the raw value. But I'll try again later

Comment (yeganer) [inline carsus/io/output/tardis_.py]: @mishinma Thanks for pointing this out, I tested your syntax and it works. I should write a short documentation about this nice feature

File: carsus/base.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def init_db(fname=None, **kwargs):
     """
     print "Initializing the database"
 
-    url = "sqlite:////" + fname if fname is not None else "sqlite://"
+    url = "sqlite:///" + fname if fname is not None else "sqlite://"
```
Comment (wkerzendorf) [inline carsus/base.py]: my suggestions is to get rid of this and just have db url. Then people can do that themselves (including different databases).

File: carsus/io/output/tardis_.py [hunk 3]
```diff
@@ -5,18 +5,41 @@
...
-from carsus.model import Atom, Ion, Line, Level, DataSource, ECollision, MEDIUM_AIR, MEDIUM_VACUUM
+from carsus.model import (
+        Atom,
+        Ion,
+        Line,
+        Level,
+        LevelEnergy,
+        DataSource,
+        ECollision,
+        MEDIUM_AIR,
```
Comment (wkerzendorf) [inline carsus/io/output/tardis_.py]: Are these enums or consts?

File: carsus/io/output/tardis_.py [hunk 4]
```diff
@@ -356,113 +379,160 @@ def lines(self):
...
-                        break
-                # If the measured energy is not available, try to get the first one
-                if energy is None:
-                    energy = lvl.energies[0].quantity
-            except IndexError:
-                print "No energy is available for level {0}".format(lvl.level_id)
-                continue
-            levels.append(
-                (lvl.level_id, lvl.atomic_number, lvl.ion_charge, energy.value, lvl.g))
+    def _get_all_levels_data(self):
```
Comment (wkerzendorf) [inline carsus/io/output/tardis_.py]: quick documentation. just a quick statement of what this does. Do this as well for the other functions.

File: carsus/io/output/tardis_.py (hunk 3 above)
Comment (yeganer) [inline carsus/io/output/tardis_.py]: I think they are constants, should they be enums?

File: carsus/base.py (hunk 2 above)
Comment (yeganer) [inline carsus/base.py]: That was my original suggestion

File: carsus/io/output/tardis_.py (hunk 4 above)
Comment (yeganer) [inline carsus/io/output/tardis_.py]: Like a docstring?

File: carsus/io/output/tardis_.py (hunk 3 above)
Comment (wkerzendorf) [inline carsus/io/output/tardis_.py]: yes a docustring - but you don't need to do the full 

```python

Parameters
------------
xxx
```

But just a quick line what this does (for the private functions).

**Pull request merged**